rules/windows/registry/registry_set/registry_set_creation_service_susp_folder.yml

```
@@ -39,7 +40,11 @@ detection:
             - '\Perflogs\'
             - '\ADMIN$\'
             - '\Temp\'
-    condition: 1 of selection_*
+    filter_1:
+        Image|contains|all:
```

Just for future reference

```suggestion
        Image|contains|all: # Filter FP with Avast software
```